BUILD

```diff
@@ -5,6 +5,20 @@ load("//rules/go:index.bzl", "go_sdk_tool")
 
 package(default_visibility = ["//visibility:public"])
 
+# This is the "go" binary from the Go SDK. It is aliased to the correct
+# binary for the current platform.
+alias(
+    name = "go_binary",
+    actual = select(
+        {
+            "@bazel_tools//src/conditions:darwin_arm64": "@go_sdk_darwin_arm64//:bin/go",
+            "@bazel_tools//src/conditions:darwin_x86_64": "@go_sdk_darwin_amd64//:bin/go",
+            "@bazel_tools//src/conditions:linux_aarch64": "@go_sdk_linux_arm64//:bin/go",
+            "@bazel_tools//src/conditions:linux_x86_64": "@go_sdk_linux_amd64//:bin/go",
+        },
```

Comment:
This could also be written as

```
        {
            "@bazel_tools//src/conditions:{}_{}".format(
                k,
                "x86_64" if v == "amd64" else "aarch64" if k == "linux" else v,
            ): "@go_sdk_{}_{}//:bin/go".format(k, v)
            for k, v in [
                (goos, goarch)
                for goos in [
                    "darwin",
                    "linux",
                ]
                for goarch in [
                    "amd64",
                    "aarch64",
                ]
            ]
        },
```

But let's not make our life harder

Reply:
Oh neat!

Do you want to merge that separately or should I pick out those hunks into this PR?

Reply:
PTAL